### ISSUE TITLE
Horizontal lines visible in output

### ISSUE THREAD

**Issue by owenthewizard** [fschutt/fastblur#1]
When I run the example code, the screenshot looks like it was blurred horizontally more so than uniformly.

**Comment by owenthewizard** [comment]
Modifying box_blur like so seems to help.

```rust
fn box_blur(backbuf: &mut Vec<[u8;3]>, frontbuf: &mut Vec<[u8;3]>, width: usize, height: usize, blur_radius: usize)
{
    box_blur_horz(backbuf, frontbuf, width, height, blur_radius); // <- second line ovverrides first line, why?
    *backbuf = frontbuf.clone();
    box_blur_vert(backbuf, frontbuf, width, height, blur_radius); // both functions should mutate the data, not clone it!
    *backbuf = frontbuf.clone();
    box_blur_horz(backbuf, frontbuf, width, height, blur_radius); // <- second line ovverrides first line, why?
    *backbuf = frontbuf.clone();
    box_blur_vert(backbuf, frontbuf, width, height, blur_radius); // both functions should mutate the data, not clone it!
}
```

**Comment by Shnatsel** [comment]
Appears to be fixed, either by 63b7013630949beddbb57b3b7474175d7ef3dede or probably even earlier.

**Comment by fschutt** [comment]
Ok, thanks for reporting
